IM_calculation/IM/im_calculation.py

```diff
@@ -110,6 +110,28 @@ def calculate_rotd(
     return value_dict
 
 
+def compute_adv_measure(waveform, advanced_im_config, output_dir):
+    """
+    Wrapper function to call advanced IM workflow
+    :param waveform: Tuple of waveform objects (first is acc, second is vel)
+    :param advanced_im_config: advanced_im_config Named Tuple containing list of IMs, config file and path to OpenSeeS
+    :param output_dir: Directory where output folders are contained. Structure is /path/to/output_dir/station/im_name
+    :return:
+    """
+
+    if (advanced_im_config is not None) and (advanced_im_config.IM_list is not None):
```

Comment:
I'm not sure brackets increase readability. Thoughts?

Reply:
I like that it breaks up the line into "if A and B", and that you can easily distinguish what A and B are. 
Without brackets is probably ok, but I'd rather leave it as is